api/src/main/openapi/paths/metrics-portfolio.yaml

```
@@ -0,0 +1,33 @@
+# This file is part of Dependency-Track.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) OWASP Foundation. All Rights Reserved.
+get:
+  operationId: getPortfolioCurrentMetrics
+  summary: Returns current metrics for the entire portfolio.
+  description: Requires permission <strong>VIEW_PORTFOLIO</strong>
+  tags:
+    - Metrics
+  responses:
+    "200":
+      description: Current metrics for the entire portfolio
+      content:
+        application/json:
+          schema:
+            $ref: "../components/schemas/portfolio-metrics-response.yaml"
+    "401":
+      $ref: "../components/responses/generic-unauthorized-error.yaml"
```

We respond with a 403 if the client is missing the necessary permissions:

```suggestion
    "401":
      $ref: "../components/responses/generic-unauthorized-error.yaml"
    "403":
      $ref: "../components/responses/generic-forbidden-error.yaml"
```